www/shared/components/contacts/Contacts.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import contacts from './Contacts.data';
```

Comment:
As a convention, I believe we shoud use `.data.js` when importing these types of files so it doesn't seem like `.data` is a valid extension.